chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md

```diff
@@ -438,15 +385,6 @@ norm_eigs.sort()
 "Norms of eigenvalues: {}".format(norm_eigs)
 ```
 
-```{.python .input}
-#@tab pytorch
-# Compute the eigenvalues
-eigs = torch.eig(A)[0][:,0].tolist()
-norm_eigs = [torch.abs(torch.tensor(x)) for x in eigs]
-norm_eigs.sort()
-"Norms of eigenvalues: {}".format(norm_eigs)
```

Comment:
Bug here

Reply:
please fix it in your next pr and double check the log generated in CI